packages/react-native/template/README.md

```diff
@@ -0,0 +1,79 @@
+This is a new [**React Native**](https://reactnative.dev) project, bootstrapped using [`@react-native-community/cli`](https://github.com/react-native-community/cli).
+
+# Getting Started
+
+>**Note**: Make sure you have completed the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup) instructions till "Creating a new application" step, before proceeding.
+
```

Comment:
we need to add an extra instruction to install `cmake` for iOS builds as it is required to build Hermes from source.

That step is not in the Environment Setup as it is not required for an app created using the RN CLI.

Suggested instructions should be:
```
brew install cmake
```